### PR TITLE
Persistence/setup UI

### LINKED ISSUE #982: Persistence/Set up UI
This issue is to set up the UI for persistence.

### PR DESCRIPTION
**Pull request by teamchong**
Closes #982

## Checklist
- [X] Ran Linting
- [X] Targeted PR against correct branch.
- [X] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [X] Added an entry to the `CHANGELOG.md` file.
- [X] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: CHANGELOG_PERSISTENCE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Unreleased
+
+## Changes
+- 
+
```
Comment (yayay927) [inline CHANGELOG_PERSISTENCE.md:5]: It's Persistence, not Assetmantle 😂
Also the issue number is not correct.

**Review by yayay927 — CHANGES_REQUESTED** [review]

File: CHANGELOG_PERSISTENCE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+# assetmantle-v1.0.0 - 2022-09-02
+
+## Changes
+- Set up UI [\#982](https://github.com/forbole/big-dipper-2.0-cosmos/issues/982))
```
Comment (yayay927) [inline CHANGELOG_PERSISTENCE.md:9]: It's Persistence, not Assetmantle 😂
Also the issue number is not correct.

**Review by yayay927 — CHANGES_REQUESTED** [review]

File: .github/workflows/docker_production.yml [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: Docker-Production
 on:
   push:
     tags:
-      - desmos-morpheus-apollo-1-v*
+      - persistence-1-v*
```
Comment (yayay927) [inline .github/workflows/docker_production.yml:6]: For the tags, our current naming convention is [chain name] + [chain id] + [v*]
So whenever we setup a new chain UI, we frontend need to check hasura first to find out the chain id.

You can find it in genesis:
`query MyQuery {
  genesis {
    chain_id
  }
}`

File: .github/workflows/docker_production.yml [hunk 4]
```diff
@@ -70,22 +68,20 @@ jobs:
...
+          DESMOS_AKASH_WEBHOOK_URL: ${{ secrets.PERSISTENCE_AKASH_WEBHOOK_URL }}
           AKASH_WEBHOOK_SECRET: ${{ secrets.AKASH_WEBHOOK_SECRET }}
         run: |
             BRANCH=${GITHUB_REF}
             WEBHOOK_URL=''
-            if [[ $GITHUB_REF == refs/tags/desmos-morpheus-apollo-1* ]]; then
-              echo 'Deploying to Desmos morpheus-apollo-1'
-              WEBHOOK_URL=$DESMOS_AKASH_WEBHOOK_URL
+            if [[ $GITHUB_REF == refs/tags/persistence-1* ]]; then
+              echo 'Deploying to Persistence persistence-1'
```
Comment (yayay927) [inline .github/workflows/docker_production.yml:83]: Same for line.82 and 83.

File: CHANGELOG_PERSISTENCE.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Unreleased
+
+## Changes
+- 
+
+# persistence-v1.0.0 - 2022-09-02
```
Comment (yayay927) [inline CHANGELOG_PERSISTENCE.md:6]: Did you release v1.0.0?
If not just add completed tasks under ```Unreleased Changes```
And add under version whenever release new version.

File: src/configs/chain_config.mainnet.json [hunk 6]
```diff
@@ -17,8 +17,8 @@
   "primaryTokenUnit": "udsm",
   "votingPowerTokenUnit": "udsm",
   "tokenUnits": {
-    "udsm": {
-      "display": "dsm",
+    "xprt": {
+      "display": "xprt",
       "exponent": 6
```
Comment (yayay927) [inline src/configs/chain_config.mainnet.json:22]: Please take a look of this document: https://forbole.atlassian.net/wiki/spaces/BD/pages/1769533/Create+new+explorer
And change line.17-21 accordingly.

File: src/configs/chain_config.mainnet.json [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
   "prefix": {
-    "consensus": "desmosvalcons",
-    "validator": "desmosvaloper",
-    "account": "desmos"
+    "consensus": "persistencevalcons",
+    "validator": "persistencevaloper",
+    "account": "persistence"
   },
   "genesis": {
     "time": "2021-07-13T08:00:00",
```
Comment (yayay927) [inline src/configs/chain_config.mainnet.json:14]: Please check the time from db and update.

**Review by yayay927 — CHANGES_REQUESTED** [review]

File: src/styles/global.css [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
   font-family: 'Hind Madurai';
-  src: url("/desmos/fonts/HindMadurai-Regular.ttf");
+  src: url("/persistence/fonts/HindMadurai-Regular.ttf");
   font-weight: 400;
 }
 
 @font-face {
   font-family: 'Hind Madurai';
-  src: url("/desmos/HindMadurai-Regular.ttf");
+  src: url("/persistence/HindMadurai-Regular.ttf");
```
Comment (yayay927) [inline src/styles/global.css:9]: Can you add  `fonts` after `persistence` to make line.9 the same as line.3? Sorry it's due to a bug from latest base release. I'll fix it soon so doesn't affect new bds.

**Review by yayay927 — CHANGES_REQUESTED** [review]

File: .github/workflows/docker_production.yml (hunk 3 above)
Comment (teamchong) [reply .github/workflows/docker_production.yml]: done

File: .github/workflows/docker_production.yml (hunk 4 above)
Comment (teamchong) [reply .github/workflows/docker_production.yml]: done

File: CHANGELOG_PERSISTENCE.md (hunk 5 above)
Comment (teamchong) [reply CHANGELOG_PERSISTENCE.md]: done

File: src/configs/chain_config.mainnet.json (hunk 7 above)
Comment (teamchong) [reply src/configs/chain_config.mainnet.json]: done

File: .github/workflows/docker_production.yml [hunk 9]
```diff
@@ -70,22 +68,20 @@ jobs:
...
+          DESMOS_AKASH_WEBHOOK_URL: ${{ secrets.PERSISTENCE_AKASH_WEBHOOK_URL }}
           AKASH_WEBHOOK_SECRET: ${{ secrets.AKASH_WEBHOOK_SECRET }}
         run: |
             BRANCH=${GITHUB_REF}
             WEBHOOK_URL=''
-            if [[ $GITHUB_REF == refs/tags/desmos-morpheus-apollo-1* ]]; then
-              echo 'Deploying to Desmos morpheus-apollo-1'
-              WEBHOOK_URL=$DESMOS_AKASH_WEBHOOK_URL
+            if [[ $GITHUB_REF == refs/tags/persistence-core-1* ]]; then
+              echo 'Deploying to Persistence persistence-core-1'
```
Comment (yayay927) [inline .github/workflows/docker_production.yml:83]: Line.83 should be `'Deploying to Persistence core-1'`

**Review by yayay927 — CHANGES_REQUESTED** [review]

File: .github/workflows/docker_production.yml (hunk 9 above)
Comment (teamchong) [reply .github/workflows/docker_production.yml]: done 🙏🏻

**Review by yayay927 — APPROVED** [review]

**Pull request merged**